src/main/java/org/jabref/logic/plaintextparser/ParserPipeline.java

```diff
@@ -0,0 +1,94 @@
+package org.jabref.logic.plaintextparser;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpEntity;
+import org.jabref.Globals;
+import org.jabref.logic.importer.ParseException;
+import org.jabref.logic.importer.fileformat.BibtexParser;
+import org.jabref.logic.net.HttpPostService;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.preferences.JabRefPreferences;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * This class is used to help making new entries faster by parsing a String.
+ * An external parser called grobid is being used for that.
+ */
+public class ParserPipeline {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ParserPipeline.class);
+    private static final String GROBID_PROCESS_CITATIONS_URL = "http://localhost:8070/api/processCitation";
```

Comment:
Interesting that you host it locally. Should be put in a README.md how to start it etc. You can add a `Dockerfile`.

There should also be a configuration (preference!) possibility to configure the URL. Maybe, there will be an online service.

Reply:
We now merged our changes that contain the already mentioned settings for the server. The user now can change the server in the advanced tab in the preferences.